File: 242.c
bool isAnagram(char * s, char * t){
    if(strlen(s) != strlen(t)) return false;

    int arr[26] = {0};

    for(int i = 0; i < strlen(s); i++){
        arr[(int)s[i]-'a']++;
        arr[(int)t[i]-'a']--;
    }
    for(int j=0; j<26;j++){
        if(arr[j]!=0) return false;
    }
    return true;

}

File: 242.cs
public class Solution {
    public bool IsAnagram(string s, string t) {
        if(s.Length != t.Length) return false;
        var symbolFrequency = new Dictionary<char, int>();
        for(int i =0; i<s.Length;i++){
            symbolFrequency.TryAdd(s[i], 0);
            symbolFrequency.TryAdd(t[i], 0);

            symbolFrequency[s[i]]++;
            symbolFrequency[t[i]]--;
        }

        return symbolFrequency.Values.All(f => f==0);
    }
}